controller/scheduler/fleet.py

```diff
@@ -377,6 +376,7 @@ def attach(self, name):
 
 CONTAINER_TEMPLATE = [
     {"section": "Unit", "name": "Description", "value": "{name}"},
+    {"section": "Service", "name": "EnvironmentFile", "value": "/etc/environment"},
```

Comment:
@carmstrong I put some more thought into this over the weekend.  Now I'm not certain I see the potential security issue you were raising.  If the generated application units pull in environment context from `/etc/app-environment`, isn't that only available to _systemd_ as it starts/manages those units?  i.e. I don't believe any of that is available to an app running in container unless it is explicitly passed in via `-e` flags in the `docker run` command.